### ISSUE TITLE
当我在运行main_landet.py文件时，出现了以下的报错

### ISSUE THREAD

**Issue by ZzhhzZ12345679** [voldemortX/pytorch-auto-drive#162]
您好，当我在运行main_landet.py文件时，出现了以下的报错：
D:\pythonProject\pytorch-auto-drive-master\pytorch-auto-drive-master\utils\models\lane_detection\laneatt.py:22: UserWarning: Can't complie line nms op for LaneATT. Set verbose=True for load in /utils/csrc/apis.py L9 for details.
D:/pythonProject/pytorch-auto-drive-master/pytorch-auto-drive-master/main_landet.py:29: UserWarning: Are you using Windows? Linux is recommended.
usage: main_landet.py [-h] [--exp-name EXP_NAME] [--device DEVICE]
                      [--workers WORKERS] [--batch-size BATCH_SIZE]
                      [--save-dir SAVE_DIR] [--val-num-steps VAL_NUM_STEPS]
                      [--world-size WORLD_SIZE] [--dist-url DIST_URL]
                      [--thresh THRESH] [--lr LR]
                      [--weight-decay WEIGHT_DECAY]
                      [--warmup-steps WARMUP_STEPS] [--epochs EPOCHS] --config
                      CONFIG
                      (--train | --test | --val | --fastval | --state STATE)
                      [--continue-from CONTINUE_FROM | --checkpoint CHECKPOINT]
                      [--mixed-precision] [--cfg-options CFG_OPTIONS]
main_landet.py: error: the following arguments are required: --config
请问如何解决这个报错，期待您的回复
